src/include/daos/btree.h

```diff
@@ -59,16 +59,31 @@ struct btr_node {
 	uint16_t			tn_flags;
 	/** number of keys stored in this node */
 	uint16_t			tn_keyn;
+	/** The last key in this node, used for lazy rebalance. */
+	uint16_t			tn_last_key;
 	/** padding bytes */
-	uint32_t			tn_pad_32;
+	uint16_t			tn_pad_16;
 	/** generation, reserved for COW */
 	uint64_t			tn_gen;
-	/** the first child, it is unused on leaf node */
-	umem_off_t			tn_child;
+	union {
+		/** the first child, it is unused on leaf node */
+		umem_off_t		tn_child;
+		/** Pointer to the next leaf node in DRAM. */
```

Comment:
Does this mean lazy rebalance is only allowed for DRAM btree?  We should check this precondition.

Reply:
It is the caller to guarantee that there will be no CPU during btree operations before rebalance done, not related with DRAM based or PMEM based. I will refresh the comment.